lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -47,24 +47,15 @@ class DOMSize extends Audit {
     };
   }
 
+
   /**
    * @param {!Artifacts} artifacts
    * @return {!AuditResult}
    */
   static audit(artifacts) {
     const stats = artifacts.DOMStats;
-
-    /**
-     * html >
-     *   body >
-     *     div >
-     *       span
-     */
-    const depthSnippet = stats.depth.pathToElement.reduce((str, curr, i) => {
-      return `${str}\n` + '  '.repeat(i) + `${curr} >`;
-    }, '').replace(/>$/g, '').trim();
-    const widthSnippet = 'Element with most children:\n' +
-        stats.width.pathToElement[stats.width.pathToElement.length - 1];
+    const depthSnippet = stats.depth.pathToElement.join(' > ');
+    const widthSnippet = stats.width.pathToElement.join(' > ');
```

Comment:
for the snippet i think we could simplify by providing the outerhtml snippet like https://github.com/GoogleChrome/lighthouse/blob/7c7695bc58fd735ac95539e1490e4657c6a457f2/lighthouse-core/gather/gatherers/accessibility.js#L76-L85

and forget this selector thing that will never scale.

i dont think this SOLVES the scaling problem, but it helps. maybe we put these snippets on a new row?

Reply:
yeah agreed this is better 👍 I think we *solve* this with a new type in fancy report land with the other page stats (# network requests, etc)

Reply:
I tried to get fancy and make it a proper node type but that looks a bit wonky
![image](https://user-images.githubusercontent.com/2301202/37549333-1f4a9b1a-293c-11e8-92d1-24640e8fc8cf.png)

Reply:
![image](https://user-images.githubusercontent.com/2301202/37549428-1a0cee86-293d-11e8-97a3-893d976cf241.png)

here's it as text which is what I'm going with for now, I expect this to change before 3.0 anyhow